src/GenFacades/GenFacades.Core/GenFacades.cs

```diff
@@ -189,7 +190,8 @@ private static void OutputFacadeToFile(string facadePath, HostEnvironment seedHo
                         // Convert from portable to windows PDBs if necessary.  If we convert
                         // set the pdbOutput path to a *.windows.pdb file so we can convert it back 
                         // to 'finalPdbOutputPath'.
-                        needsConversion = ConvertFromPortableIfNecessary(facade.Location, ref pdbLocation);
+                        if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                            needsConversion = ConvertFromPortableIfNecessary(facade.Location, ref pdbLocation);
```

Comment:
Style guides usually say to include braces, but I see this is done elsewhere in the file too so no big deal.